assets/training/general/environments/acpt-pytorch-2.1-cuda12.1/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+azure-ai-ml==1.2.0
+azure.identity==1.10.0
```

Comment:
(nit as this is for testing) 
Should these be updated
azure-ai-ml==1.13.0
azure-identity==1.15.0